### PR TITLE
[CONFIG] [SWE-1602] Change vars naming; change hex to rgba; add class for senseteq #done

### PR DESCRIPTION
**Pull request by AnastasiaShapowalova**

### CODE REVIEW

File: src/styles/vars/theme.light.css [hunk 1]
```diff
@@ -1,97 +1,69 @@
...
+  --gray-lighten3: rgba(230, 230, 230, 1);
 
-  /* Singletons */
-  --brandbook-module-black: #000000;
-  --brandbook-module-white: #ffffff;
   /* Inverse */
-  --brandbook-module-white-inverse: #ffffffb8;
-  --brandbook-module-neutral-inverse: #253037a3;
+  --white-inverse: rgba(255, 255, 255, 0.64);
+  --neutral-inverse: rgba(37, 48, 55, 0.64);
```
Comment (BogdanovJn) [inline src/styles/vars/theme.light.css:64]: Same to --white-inverse

File: src/styles/vars/input.css [hunk 2]
```diff
@@ -1,56 +1,36 @@
...
+    --neutral-lighten-t1
   ); /* opacity: 0.4 */
 
   /* STATES */
-  --brandbook-module-brand-input-hover-box-shadow: 0px 0px 0px 2px
-    var(--brandbook-module-brand-neutral-lighten-t2); /* opacity: 0.16*/
-  --brandbook-module-brand-input-active-box-shadow: 0px 0px 0px 2px
-    var(--brandbook-module-brand-primary);
+  --input-hover-box-shadow: 0px 0px 0px 2px var(--neutral-lighten-t2); /* opacity: 0.16*/
+  --input-active-box-shadow: 0px 0px 0px 2px var(--primary);
```
Comment (BogdanovJn) [inline src/styles/vars/input.css:21]: Looks like mistake, I can't find variable "--primary"

File: src/styles/vars/header.css [hunk 3]
```diff
@@ -1,13 +1,11 @@
...
-  --brandbook-module-header-line-height: var(--brandbook-module-text-body-lh);
-  --brandbook-module-header-font-family: var(
-    --brandbook-module-text-font-family
-  );
-  --brandbook-module-header-text-white: var(--brandbook-module-white);
-  --brandbook-module-header-text-default: var(--brandbook-module-neutral);
+  --header-font-size: var(--text-body-fs);
+  --header-line-height: var(--text-body-lh);
+  --header-font-family: var(--senseteq-text-font-family);
+  --header-text-white: var(--senseteq-white);
```
Comment (BogdanovJn) [inline src/styles/vars/header.css:9]: What the difference between "--white" and "--senseteq-white"?

File: src/styles/index.css [hunk 4]
```diff
@@ -24,3 +24,93 @@ body {
...
+  --senseteq-text-body1-fs: 1rem;
+  --senseteq-text-body2-fs: 0.875rem;
+  /* Line height */
+  --senseteq-text-subtitle1-lh: 4rem;
+  --senseteq-text-subtitle2-lh: 2rem;
+  --senseteq-text-body1-lh: 1.5rem;
+  --senseteq-text-body2-lh: 1.25rem;
+  /* Font weight */
+  --senseteq-text-fw-bold: 700;
+  --senseteq-text-fw-semi-bold: 600;
```
Comment (BogdanovJn) [inline src/styles/index.css:95]: semibold

File: src/styles/index.css [hunk 5]
```diff
@@ -24,3 +24,93 @@ body {
...
+
+.senseteq {
+  /* THEME */
+  /* Colors */
+  /* Primary */
+  --senseteq-accent: rgba(45, 184, 103);
+  /* Neutral (Dark) */
+  --senseteq-neutral: rgba(37, 48, 55);
+  /* Singletons */
+  --senseteq-white: rgb(255, 255, 255);
```
Comment (BogdanovJn) [inline src/styles/index.css:36]: Inconsistency with "--senseteq-accent", "--senseteq-neutral" and "--senseteq-white" (
find the error yourself)

File: src/app/domains/Iconography/components/IconView/IconView.styled.js [hunk 6]
```diff
@@ -4,7 +4,7 @@ const IconViewStyled = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
-  width: var(--brandbook-module-icon-view-width);
+  width: var(--icon-view-width);
```
Comment (BogdanovJn) [inline src/app/domains/Iconography/components/IconView/IconView.styled.js:7]: Not sure with width, check grid in the Figma

File: src/styles/vars/theme.light.css [hunk 7]
```diff
@@ -1,97 +1,69 @@
...
+  --neutral-lighten-t5: rgba(var(--neutral) var(--alpha-5)); /* opacity: 0.10 */
+  --neutral-lighten-t6: rgba(var(--neutral) var(--alpha-6)); /* opacity: 0.05 */
 
-  /* Success */
-  --brandbook-module-success-darken: #15803d;
-  --brandbook-module-success: #22c55e;
-  --brandbook-module-success-lighten: #86efac;
+  /* Singletons */
+  --black: rgba(0, 0, 0, 1);
+  --white: rgb(255, 255, 255);
```
[the remark this answers is not in the document]
Comment (AnastasiaShapowalova) [reply src/styles/vars/theme.light.css:35]: When we make shades we add alpha as a var(). I thought that we should write the main coulors as RGB in this case. Isn't that the point?

File: src/app/components/elements/Chapter/Chapter.styled.js [hunk 8]
```diff
@@ -1,10 +1,10 @@
...
 
 const ChapterStyled = styled.div`
-  padding-top: var(--brandbook-module-chapter-padding-y);
-  padding-bottom: var(--brandbook-module-chapter-padding-y);
-  background-color: var(--brandbook-module-chapter-bg);
-  border-bottom: 1px solid var(--brandbook-module-white);
+  padding-top: var(--chapter-padding-y);
+  padding-bottom: var(--chapter-padding-y);
+  background-color: var(--chapter-bg);
+  border-bottom: 1px solid var(--white);
```
Comment (alexpassss) [inline src/app/components/elements/Chapter/Chapter.styled.js:7]: Borders also should be moved to the base vars

File: src/app/components/elements/Divider/Divider.styled.js [hunk 9]
```diff
@@ -1,35 +1,33 @@
 import styled, { css } from 'styled-components'
 const bgColorMap = {
   accent: css`
-    background-color: var(--brandbook-module-divider-accent-bg);
+    background-color: var(--divider-accent-bg);
```
Comment (alexpassss) [inline src/app/components/elements/Divider/Divider.styled.js:4]: Same issue with objects

**Review by alexpassss — CHANGES_REQUESTED** [review]

**Pull request closed without merging**